ui/src/app/pages/access/access-renewal-page.tsx

```diff
@@ -169,15 +224,34 @@ export const AccessRenewalPage = fp.flow(
   const [publications, setPublications] = useState<boolean>(null);
   const noReportId = useId();
   const reportId = useId();
+  const [reloadDisabled, setReloadDisabled] = useState(true);
+  const [busy, setBusy] = useState(false);
```

Comment:
We typically name the spinner state "loading" ... but I'm fine with busy

Reply:
Changed for consistency